src/ralph/backends/database/es.py

```diff
@@ -62,15 +61,13 @@ def __init__(
         self.op_type = op_type
 
     def get(self, chunk_size=500):
-        """Get index documents and stream them"""
+        """Gets index documents and yields them."""
 
         for document in scan(self.client, index=self.index, size=chunk_size):
-            sys.stdout.buffer.write(
-                bytes(json.dumps(document.get("_source")) + "\n", encoding="utf-8")
-            )
+            yield document
 
     def to_documents(self, stream, get_id):
```

Comment:
Should we also type this function signature?

Reply:
And also the type of yielded objects?

Reply:
Good Idea) Thanks) Added the type hints)